library/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     compile project(':third_party:disklrucache')
     compile project(':annotation')
     compile "com.android.support:support-v4:${SUPPORT_V4_VERSION}"
-
+    compile "com.android.support:appcompat-v7:${SUPPORT_V7_VERSION}"
```

Comment:
Yeah I'm not 100% sure it makes sense to add a hard dependency on v7 either. Maybe this should be an integration library? You can probably override the setPlaceholder methods with a GlideExtension, though it's a bit weird.

As an alternative Apps can also pass in Drawables directly, so those that are using v7 can call this method themselves.